spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-core.spec.js

```diff
@@ -14,9 +14,10 @@ describe('GA4 core', function () {
   it('loads the GTM snippet', function () {
     GOVUK.analyticsGA4.core.load()
 
-    expect(window.dataLayer.length).toEqual(1)
+    expect(window.dataLayer.length).toEqual(2)
     expect(Object.keys(window.dataLayer[0])).toContain('gtm.start')
     expect(Object.keys(window.dataLayer[0])).toContain('event')
+    expect(Object.keys(window.dataLayer[1])).toContain('gtm.blocklist')
```

Comment:
Not a blocker to merging - I wonder if we should check the blocklist array entries? Just to make sure some doesn't disappear.

Reply:
Yep, sure, have added 👍

Reply:
Yup - exactly - thanks 👍